include/tcframe/spec/variable/Variable.hpp

```diff
@@ -76,6 +76,11 @@ class Variable {
             throw runtime_error("Cannot parse for " + context + ". Found: '" + found + "'");
         }
     }
+
+    static void parseRawLine(istream* in, string& val) {
+        getline(*in, val);
+        in->unget();
```

Comment:
Oh, you're right; I just manually tested it, and 'o' will be "unget" -_- This is because C++'s getline() can accept either EOF or newline as the delimiter. Fortunately, it currently does not matter because error will be thrown anyway (expecting '\n').

Anyway, I'll change it so that it will unget only if not EOF.

We should then test the next character in the stream after every parse for safety, but since it will touch many parsing tests, I'll do that in a separate PR.